### PR TITLE
Move Instrument Locations to Senaite setup folder

### PR DESCRIPTION
**Pull request by Bugerman58**
## Description of the issue/feature this PR addresses

Move InstrumentLocation and InstrumentLocations to Senaite setup folder, cleanup and refactoring

## Current behavior before PR

Now it lives in bika_setup folder

## Desired behavior after PR is merged

Now it lives in setup folder

--
I confirm I have tested this PR thoroughly and coded it according to [PEP8][1]
and [Plone's Python styleguide][2] standards.

[1]: https://www.python.org/dev/peps/pep-0008
[2]: https://docs.plone.org/develop/styleguide/python.html

### CODE REVIEW

**Comment by Bugerman58** [comment]
@xispa Hi, this PR ready for review. pls, check it.

**Comment by ramonski** [comment]
Hi @Bugerman58, Thanks for keeping this one in sync with the latest PR merges in 2.x!
This one will be next, promise!

**Comment by Bugerman58** [comment]
> Hi @Bugerman58, Thanks for keeping this one in sync with the latest PR merges in 2.x! This one will be next, promise!

Hi, @ramonski Thanks! =)

**Comment by ramonski** [comment]
Hi @Bugerman58,
The migrated instrument location has duplicate Title/Description fields:

Workbench 1

Probably, you need to remove the `plone.app.dexterity.behaviors.metadata.IBasic` behavior from the `InstrumentLocation.xml`...

**Comment by ramonski** [comment]
Yes, removing it manually also removes the default Title/Description coming in from the behavior:

Instrument Location 2024-06-16 2 PM-05-52

Workbench 1

File: src/senaite/core/upgrade/v02_06_000.zcml [hunk 1]
```diff
@@ -6,9 +6,17 @@
   <genericsetup:upgradeStep
       title="SENAITE.CORE 2.6.0: Move Instrument Locations to Senaite setup"
       description="Move Instrument Locations to Senaite setup"
+      source="2636"
+      destination="2637"
+      handler=".v02_06_000.move_isntrumentlocations"
```
Comment (ramonski) [inline src/senaite/core/upgrade/v02_06_000.zcml]: small typo here

Comment (Bugerman58) [reply src/senaite/core/upgrade/v02_06_000.zcml]: :) sorry

**Review by ramonski — APPROVED** [review]
Great, thanks @Bugerman58 for your work!

**Pull request merged**